cli/report/report.py

```diff
@@ -167,6 +200,9 @@ def file_jira_issues(
                 logs_dir=job.logs_dir,
                 junit_dir=job.junit_dir,
                 junit_file=pair["failure"].failed_test_junit_path if firewatch_config.verbose_test_failure_reporting else None,  # type: ignore
+                is_gitleaks_failure=True
```

Comment:
I don't think I see this being used in the for loop. I could be missing it though.

Reply:
I had to double-check this myself, because the Black re-formatting "hides" [the rest of the ternary statement](https://github.com/amp-rh/firewatch/blob/878610b2dba34b22e0d3b249b9891bf336bc86fc/cli/report/report.py#L203-L205) at first glance. I will probably change this for readability, knowing that.

This is used to [identify gitleaks failures in the loop](https://github.com/amp-rh/firewatch/blob/878610b2dba34b22e0d3b249b9891bf336bc86fc/cli/report/report.py#L537-L543C31) and handle adding attachments to jira issues differently from other failure types, since this could not be determined from the other method inputs.

Reply:
Thank you for pointing that out!